src/_scss/pages/search/results/visualizations/geo/map/_toggle.scss

```diff
@@ -80,12 +80,48 @@
         .map-layer__cd-tooltip {
             display: inline-flex;
             white-space: normal;
-            .tooltip-spacer {
-                max-width: 316px !important;
-                @media (max-width: $tablet-screen) {
-                    left: rem(-19) !important;
+
+            .state-profile__map-toggle {
+                .tooltip-spacer {
+                    width: 316px !important;
+                    @media (max-width: 767px) {
+                        top: rem(62) !important;
+                        left: rem(-30) !important;
+                    }
+                    @media ((min-width: $tablet-screen) and (max-width: $medium-screen)) {
+                        top: rem(40) !important;
+                        left: rem(-30) !important;
+
+                        .smart-bottom-right {
```

Comment:
I don't see where this class is ever added to the parent class. Is it being used?

Reply:
this is nested in the tooltip wrapper it is for the pointer, it's there